storm-core/src/clj/backtype/storm/daemon/supervisor.clj

```diff
@@ -31,6 +32,7 @@
 
 (defmulti download-storm-code cluster-mode)
 (defmulti launch-worker (fn [supervisor & _] (cluster-mode (:conf supervisor))))
+(defmulti mk-bt-tracker cluster-mode)
```

Comment:
Should this just be removed? we are still downloading from nimbus, so this seems like it is not needed here.